src/bin/tikv-ctl.rs

```diff
@@ -118,7 +118,7 @@ fn main() {
         dump_range(db, from, to, limit, cf_name);
     }
     else {
-        panic!("The command is not supported now.");
+        let _ = app.print_help();
```

Comment:
Yep, it will print the help message. No other code after it so the program will return.